### PR TITLE
Updates to Changelog, RTD and version for the 0.7.3-RC0 release

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

**Comment by swcurran** [comment]
@andrewwhitehead -- this is ready to go.

**Comment by swcurran** [comment]
And not ready to go...pending a change from @shaangill025 about #1553 .

Please let me know when #1553 is ready to go and we can release.

Thanks!

File: aries_cloudagent/version.py [hunk 1]
```diff
@@ -1,3 +1,3 @@
 """Library version information."""
 
-__version__ = "0.7.2"
+__version__ = "v0.7.3-rc0"
```
Comment (andrewwhitehead) [inline aries_cloudagent/version.py:3]: No 'v' here 😲

Comment (swcurran) [reply aries_cloudagent/version.py]: Fixed in docs and in code locations.  Doh.

**Review by andrewwhitehead — APPROVED** [review]

**Pull request merged**